src/luascript.cpp

```diff
@@ -9727,7 +9728,7 @@ int LuaScriptInterface::luaPlayerSetBankBalance(lua_State* L)
 
 int LuaScriptInterface::luaPlayerGetStorageValue(lua_State* L)
 {
-	// player:getStorageValue(key)
+	// player:getStorageValue(key[, default = -1])
```

Comment:
https://github.com/otland/forgottenserver/pull/4298#discussion_r1088247646

:)

Reply:
OMG thanks for reminding me, I already changed it.
That's what happens to me for walking sleepy at dawn doing these things hahaha